soc/nordic/Kconfig

```diff
@@ -75,28 +75,6 @@ config NRF_ACL_FLASH_REGION_SIZE
 	help
 	  FLASH region size for the NRF_ACL peripheral.
 
-config NFCT_PINS_AS_GPIOS
```

Comment:
Remove also this line: https://github.com/zephyrproject-rtos/zephyr/blob/4852719f98b3bac9c05196d9855d897f47e3c82f/modules/hal_nordic/nrfx/CMakeLists.txt#L180